roles/kubevirt-resources/tasks/provision.yaml

```diff
@@ -0,0 +1,37 @@
+- name: Login As Super User
+  command: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  when: cluster=="openshift"
+        and admin_user is defined
+        and admin_password is defined
+
+- name: Create {{ namespace }} namespace
+  shell: kubectl create namespace {{ namespace }}
+  ignore_errors : true
```

Comment:
why do you ignore an error in this particular task?

Reply:
We don't want this to fail if the namespace already exists because it doesn't change the outcome of the playbook. In addition, some deployments of k8s by default create the kube-system namespace, which would break this task.

Reply:
So you can check if it exists, if not create it.